recipes-debian/gnutls/gnutls_debian.bbappend

```diff
@@ -0,0 +1,2 @@
+SHRT_VER = "${@d.getVar('PV').split('.')[0]}.${@d.getVar('PV').split('.')[1]}"
```

Comment:
In [poky](https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-support/gnutls/gnutls_3.6.7.bb?h=warrior&id=09dea32f477a3e8855bd35af1c042004a823de8f), SHRT_VER is used for url to source tar file.

```
SHRT_VER = "${@d.getVar('PV').split('.')[0]}.${@d.getVar('PV').split('.')[1]}"

SRC_URI = "https://www.gnupg.org/ftp/gcrypt/gnutls/v${SHRT_VER}/gnutls-${PV}.tar.xz \
           file://arm_eabi.patch \
"
```

However, meta-debian's recipe doesn't use it. so I think remove .bb file and .inc file is enough.